app/models/visit.rb

```diff
@@ -154,7 +154,7 @@ def last_visit_state
   end
 
   def additional_visitors
-    @additional_visitors ||= visitors.reject { |v| v == principal_visitor }
+    @additional_visitors ||= visitors.reject { |v| v == lead_visitor }
```

Comment:
Don't think it's more readable and in terms of efficiency it's not sure. It's more concise though